Firestore/core/src/firebase/firestore/local/local_view_changes.h

```diff
@@ -63,6 +69,7 @@ class LocalViewChanges {
 
  private:
   model::TargetId target_id_;
+  bool from_cache_;
```

Comment:
This should have an initializer (as should `target_id_` above it).

Reply:
Done (here and elsewhere)